tests/integration/questionnaire/test_questionnaire_last_viewed_guidance.py

```diff
@@ -0,0 +1,286 @@
+import random
+import string
+from tests.integration.integration_test_case import IntegrationTestCase
+
+
+class TestLastViewedGuidance(IntegrationTestCase):
+    def test_last_viewed_question_guidance_not_shown_on_survey_launch(self):
+        # Given
+        self.launchSurvey("test_last_viewed_question_guidance")
+
+        # Then last viewed question guidance should not be shown
+        self._last_viewed_question_guidance_not_shown_assertion()
```

Comment:
The method name should be consistent with the generic asserts and start with assert e.g. `_assert_last_viewed_question_guidance_not_shown`